### PR TITLE
yaqd-control

### PR DESCRIPTION
**Pull request by untzag**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ksunden** [comment]
Yes, I am willing to maintain

**Comment by untzag** [comment]
This package vendors [nssm](https://nssm.cc/), which is public domain. I'd really appreciate some advice about how to proceed. I would be willing to maintain a separate recipe if that seems appropriate?

**Comment by untzag** [comment]
@conda-forge/help-python could I get a review please?

File: recipes/yaqd-control/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3e292ee3d55fd310dc58d9ae5d7d4736ebf924cc428d376988ed46f48fa594ce
+
+build:
+  skip: true  # [py<37]
```
Comment (xhochy) [inline recipes/yaqd-control/meta.yaml:13]: This can be noarch.
```suggestion
  noarch: python
```

File: recipes/yaqd-control/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  entry_points:
+    - yaqd=yaqd_control.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (xhochy) [inline recipes/yaqd-control/meta.yaml:21]:
```suggestion
    - python >=3.7
```

File: recipes/yaqd-control/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - pip
+  run:
+    - appdirs
+    - click
+    - colorama
+    - msgpack-python >=1.0.0
+    - prettytable
+    - python
```
Comment (xhochy) [inline recipes/yaqd-control/meta.yaml:29]:
```suggestion
    - python >=3.7
```

**Comment by untzag** [comment]
thanks for your comments @xhochy

**Pull request merged**